tools/defaults.bzl

```diff
@@ -1,9 +1,18 @@
 load("@npm//@bazel/jasmine:index.bzl", _jasmine_node_test = "jasmine_node_test")
-load("@npm//@bazel/typescript:index.bzl", _ts_project = "ts_project")
+load("@npm//@bazel/typescript:index.bzl", _ts_library = "ts_library", _ts_project = "ts_project")
+
+def jasmine_node_test(name, specs = [], **kwargs):
+    native.filegroup(
+        name = "%s_js_files",
```

Comment:
Why is the filegroup needed? The Jasmine rule would automatically extract specs from the deps. Is there a specific use-case?

Reply:
Because we have set `use_direct_specs` to be `True`, we have to provide the specific spec javascript files as srcs in `jasmine_node_test`